router/registry_handler_test.go

```diff
@@ -0,0 +1,46 @@
+package router
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestRegistry(t *testing.T) {
+	r := NewTestRouter(withLiquid)
+	resp := mintRequest(r, liquidAddress, assetQuantity, name, ticker)
+	resp = registryRequest(r, []interface{}{})
+
+	if resp.Code != http.StatusOK {
+		t.Fatalf("Expected status %d, got: %d\n", http.StatusOK, resp.Code)
+	}
+
+	list := []interface{}{}
+	json.Unmarshal(resp.Body.Bytes(), &list)
+
+	if len(list) < 1 {
```

Comment:
Would be nice check the entry to test database retrieval too

Reply:
Do you mean testing the electrs `/asset/:asset_id` endpoint?

Reply:
No, just the database retrieval matches the one has ben stored by the mint request